.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "stable"
+install:
+  - npm install
+script:
+  - npm run build 
```

Comment:
Now it seems that test are not runned.
Can you add `npm test`

Reply:
Added :)